blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/MultiBranchPipelineQueueContainer.java

```diff
@@ -36,13 +36,19 @@ public BlueQueueItem get(String name) {
             if(item != null){
                 PipelineImpl pipeline = (PipelineImpl) multiBranchPipeline.getBranches().get(item.task.getOwnerTask().getName());
                 if(pipeline != null) {
-                    int runId=0;
+
                     if(item.task instanceof ExecutorStepExecution.PlaceholderTask) {
                         ExecutorStepExecution.PlaceholderTask task = (ExecutorStepExecution.PlaceholderTask) item.task;
-                        runId = task.run().getNumber();
+                        int runNumber;
+                        if(task.run() == null){
+                            runNumber = pipeline.job.getNextBuildNumber();
```

Comment:
Thats not quite right. What we actaully need to get the list of queued items and figure out what number it is for that job. But I think this is fine and I'll add this with my incoming replay PR